openupgradelib/openupgrade_160.py

```diff
@@ -466,6 +467,24 @@ def fill_analytic_distribution(
         env.cr,
         f"ALTER TABLE {table} ADD COLUMN IF NOT EXISTS {column} jsonb",
     )
+    query_union = ""
+    if m2m_rel:
```

Comment:
```suggestion
    if m2m_rel and m2m_column1:
```

Reply:
Well, it's your responsibility to call correctly the method if you want to process analytic tags. The other parameter is the control one, but OK, modified to your taste.